spec/helpers/admin/users_helper_spec.rb

```diff
@@ -1,6 +1,33 @@
 require "rails_helper"
 
 describe Admin::UsersHelper do
+  describe "#role_options" do
+    let(:user) { create(:user) }
+
+    it "returns base roles" do
+      user.add_role(:admin)
+      roles = helper.role_options(user)
+      expect(roles).to have_key("Base Roles")
+      expect(roles["Base Roles"]).to eq Constants::Role::BASE_ROLES
+    end
+
+    it "returns special roles" do
```

Comment:
Quick, non-blocking nit-pick (same as above!): could we please aggregate the failures here (and in the tests below) since we have multiple expectations within this single block?
```suggestion
    it "returns special roles", :aggregate_failures do
```